retorchfiles/scripts/coilifecycles/coi-setup.sh

```diff
@@ -5,13 +5,13 @@ SEL_LOG_DIR="/opt/selenoid/logs/"
 
 # Function to remove older files
 remove_old_files() {
-    dir=$1
-    days=$2
-    echo "Removing older files ($days days) in $dir! The number of files prior to remove:"
-    find "$dir" | wc -l
-    find "$dir" -mindepth 1 -maxdepth 1 -mtime +$days -exec rm -rf {} \;
-    echo "Removing older files ($days days) in $dir! The number of files after remove:"
-    find "$dir" | wc -l
+  dir=$1
+  days=$2
+  echo "Removing older files ($days days) in $dir! The number of files prior to remove:"
+  find "$dir" | wc -l
+  find "$dir" -mindepth 1 -maxdepth 1 -mtime +$days -exec rm -rf {} \;
+  echo "Removing older files ($days days) in $dir! The number of files after remove:"
+  find "$dir" | wc -l
```

Comment:
Remove formatting changes here and the next changed files (see https://github.com/giis-uniovi/.github/blob/main/profile/CONTRIBUTING.md, third item on PR).